app/styles/_layout.scss

```diff
@@ -108,7 +108,7 @@
  *          p Just some text in a medium width wrapper
  */
 .l-medium {
-    width: 100%;
+    width: 90%;
```

Comment:
Yeah. So when the screen width is => 800px, width of medium will be 800px. But it's < 800px, it becomes 90% of the screen. It's a way to handle mobile.